src/test/appVersions.test.js

```diff
@@ -181,34 +204,30 @@ describe("App Version List component", function () {
       "2015-06-29T13:02:19.363Z"
     ];
 
-    this.renderer = TestUtils.createRenderer();
-
-    this.renderer.render(<AppVersionListComponent
-      appId={"/app-test"} />);
-
-    this.component = this.renderer.getRenderOutput();
+    this.component = mount(<AppVersionListComponent appId={"/app-test"} />);
   });
 
-  afterEach(function () {
-    this.renderer.unmount();
+  after(function () {
+    this.component.component.getInstance().componentWillUnmount();
```

Comment:
@aldipower Do you think about something like `TestUtil.getComponentInstance(this.component)`?